### ISSUE TITLE
whalebrew install returns client version error

### ISSUE THREAD

**Issue by jimscard** [whalebrew/whalebrew#279]
Several whalebrew commands return an error message:

```
└─$ whalebrew lint whalebrew/figlet
failed to inspect docker image: Error response from daemon: client version 1.20 is too old. Minimum supported API version is 1.24, please upgrade your client to a newer version
```

```
└─$ whalebrew install jq
failed to inspect docker image: Error response from daemon: client version 1.20 is too old. Minimum supported API version is 1.24, please upgrade your client to a newer version
```

Already installed whalebrew apps work, but can't install any new ones....

**Comment by thaJeztah** [comment]
What version do you have installed? I see the docker sdk was updated recently (and part of the latest tag / release in this repo)

**Comment by jimscard** [comment]
I just did a `brew reinstall whalebrew` just to make sure the issue still repo'd .. 

```
└─$ whalebrew version
Whalebrew 0.4.1+homebrew
```

**Comment by thaJeztah** [comment]
Oh! I just notice it may be shelling out to the docker cli (sorry, not really familiar with the source code); what version of docker are you running? (full output of `docker version`)?

**Comment by jimscard** [comment]
```
└─$ docker version
Client:
 Cloud integration: v1.0.35+desktop.11
 Version:           25.0.3
 API version:       1.44
 Go version:        go1.21.6
 Git commit:        4debf41
 Built:             Tue Feb  6 21:13:26 2024
 OS/Arch:           darwin/amd64
 Context:           desktop-linux

Server: Docker Desktop 4.28.0 (139021)
 Engine:
  Version:          25.0.3
  API version:      1.44 (minimum version 1.24)
  Go version:       go1.21.6
  Git commit:       f417435
  Built:            Tue Feb  6 21:14:25 2024
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.28
  GitCommit:        ae07eda36dd25f8a1b98dfbf587313b99c0190bb
 runc:
  Version:          1.1.12
  GitCommit:        v1.1.12-0-g51d5e94
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**Comment by jimscard** [comment]
Ok, figured it out based on your docker clue.  I applied PR #278 and built locally, and the resultant binary worked.  I tried triggering a release via github actions on my fork, btw, and it failed because of node 16 dependent actions - I wonder if that's why there are approved pull requests, but no new release here?

**Comment by 13steinj** [comment]
At the risk of potentially being blunt-- is this project unmaintained / in need of another maintainer?

**Comment by moonfruit** [comment]
Same problem here

**Comment by rbe** [comment]
Same here. Worked with an existing installation of Docker Desktop, but stopped after installing Docker via Homebrew as shown in README.

**Comment by 13steinj** [comment]
FYI @rbe (I'm also using homebrew, but not for docker) in my case I worked around the issue via...

```
brew tap --force homebrew/core
brew edit whalebrew
```
added `system "git", "merge", "origin/dependabot/go_modules/github.com/docker/cli-25.0.1incompatible"` to the beginning of the install function

```
HOMEBREW_NO_INSTALL_FROM_API=1 brew install -s whalebrew
```

**Comment by Sam-Bate-ITV** [comment]
@bfirsh any chance of getting https://github.com/whalebrew/whalebrew/pull/278 merged and a new tag?

**Comment by Darkflib** [comment]
> FYI @rbe (I'm also using homebrew, but not for docker) in my case I worked around the issue via...
> 
> ```
> brew tap --force homebrew/core
> brew edit whalebrew
> ```
> 
> added `system "git", "merge", "origin/dependabot/go_modules/github.com/docker/cli-25.0.1incompatible"` to the beginning of the install function
> 
> ```
> HOMEBREW_NO_INSTALL_FROM_API=1 brew install -s whalebrew
> ```

Worked for me after I fixed an issue with brew auto updating and caused a merge conflict.

Thanks

**Comment by moonfruit** [comment]
I note that #298 has been merged, so only one question remains: when will the new version be released? It's been 10 months since the last release.

**Comment by 13steinj** [comment]
From a homebrew perspective, since it's been merged, I _think_ one can just `brew install whalebrew --HEAD` now.

**Comment by tjamet** [comment]
Hi!

Most of the pending PRs have been merged and [0.5.0](https://github.com/whalebrew/whalebrew/releases/tag/0.5.0) has been released.

This should fix this issue.

I will close it. Please feel free to re-open it if you still face difficulties
